Games/types/Mafia/roles/Village/President.js

```diff
@@ -4,6 +4,26 @@ module.exports = class President extends Role {
   constructor(player, data) {
     super("President", player, data);
     this.alignment = "Village";
-    this.cards = ["VillageCore", "WinWithVillage", "KillAlignedOnDeath"];
+    this.cards = ["VillageCore", "WinWithVillage",];
+    this.listeners = {
+      roleAssigned: function (player) {
```

Comment:
is the purpose of this just to announce the president's modifiers? r9 made a function that can be used here

Reply:
yes, let's use R9's instead then